scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -115,12 +120,93 @@ def strip_NaNs(output_workspace, base_output_name):
 
     return base_output_name
 
+##############################################################################
+# 26 March 2019
```

Comment:
Please remove dates in comments, Git can be used to find out when things were done.